### ISSUE TITLE
Unable to test demo template pops an error

### ISSUE THREAD

**Issue by cconstab** [atsign-foundation/at_app_cli#46]
**Describe the bug**
Unable to test demo template pops an error

**To Reproduce**
1. Follow instructions in https://docs.google.com/document/d/1XxJ39T40h7flefqgK2zYZCH4RgvcOH13Pfm1PSoH1nA/edit
2. Then ran testing command and it failed

```
PS C:\Users\colin\GitHub\@foundation\at_app> dart run at_app/bin/at_app.dart create -d snackbar --local s:\TEMP\test   
Creating project in s:\TEMP\test

Project created, now adding a little @ magic...
[!] ERROR: An unknown issue occurred:

Bad state: No element

Please file a ticket to prevent this from happening again:
https://github.com/atsign-foundation/at_app
PS C:\Users\colin\GitHub\@foundation\at_app

```

**Expected behavior**
Template to create runabable app from template

**Screenshots**
See above
**Smartphone (please complete the following information):**
 - Windows 
```
PS C:\Users\colin\GitHub\@foundation\at_app> flutter --version
Flutter 2.8.0-3.2.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 2901cd7208 (4 weeks ago) • 2021-11-18 11:15:09 -0800
Engine • revision bcc2b7f12c
Tools • Dart 2.15.0 (build 2.15.0-268.18.beta)
PS C:\Users\colin\GitHub\@foundation\at_app> 
```

**Additional context**
If you need to I can create a branch so you can see my attempt of creating the templates as this could well be me!

**Comment by xavierchanth** [comment]
Yes, please upload the branch.

**Comment by xavierchanth** [comment]
Fixed in at_onboarding_flutter 3.1.2.
